commons_api/settings.py

```diff
@@ -108,6 +110,8 @@
 
 CELERY_RESULT_BACKEND = 'django-db'
 
+BOUNDARIES_SIMPLE_SHAPE_TOLERANCE = 0.001
```

Comment:
Another variable one to be made environment

Reply:
Done :-)